topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,299 @@
+>       {: .comment}
+> 2. **hicBuildMatrix** {% icon tool %}: Re-run hicBuildMatrix on `SRR1956528_1.sam` and `SRR1956528_2.sam`, `SRR1956529_1.sam` and `SRR1956529_2.sam` respectively.
+>
+>    > ### {% icon question %} Question
+>    >
+>    > How many selected reads do the output bam files have?
+>    >
+>    > <details>
+>    > <summary>Click to view answers</summary>
+>    > The output bam files show that we have around 34M, 54M and 58M selected reads for `SRR1956527`, `SRR1956528` & `SRR1956529`, respectively.
+>    > </details>
+>    {: .question}
+>
+>       > ### {% icon comment %} Comment
+>       >
+>       > Normally 25% of the total reads are selected. The output matrices have counts for the genomic regions. The extension of output matrix files is .h5.
+>       > A quality report is created in e.g. `hicMatrix/SRR1956527_10kb_QC`, have a look at the report hicQC.html.
+>       {: .comment}
+>
+> 3. **hicSumMatrices** {% icon tool %}: Run hicSumMatrices on `.h5` outputs of hicBuildMatrix tool with default settings.
+>
+{: .hands_on}
+
+# Plotting the Hi-C matrix
+
+A 10kb bin matrix is quite large to plot and is better to reduce the resolution (to know the size of a Hi-C matrix use the tool [hicInfo](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicInfo.html#hicinfo)), i.e. we usually run out of memory for a 1 kb or a 10 kb matrix and second, the time to plot is very long (minutes instead of seconds). For this we use the tool [hicMergeMatrixBins](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicMergeMatrixBins.html#hicmergematrixbins).
+
+[hicMergeMatrixBins](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicMergeMatrixBins.html#hicmergematrixbins) merges the bins into larger bins of given number (specified by –numBins). We will merge 1000 bins in the original (uncorrected) matrix and then correct it. The new bin size is going to be 10.000 bp * 100 = 1.000.000 bp = 1 Mb
+
+> ### {% icon hands_on %} Hands-on: hicMergeMatrixBins
+>
+> 1. **hicMergeMatrixBins** {% icon tool %}: Run hicMergeMatrixBins on the output from previous step setting the following parameter:
+>    - "Number of bins to merge" to `100`
+>
+> 2. **hicPlotMatrix** {% icon tool %}: Run hicPlotMatrix on the output from hicMergeMatrixBins adjusting the parameters:
+>    - "Plot title" to `Hi-C matrix for mESC`
+>    - "Plot per chromosome" to `True`
+>    - "Chromosomes to include (and order to plot in)" to `chr1 chr2 chr3 chr4 chr5 chr6 chr7 chr8 chr9 chr10 chr11 chr12 chr13 chr14 chr15 chr16 chr17 chr18 chr19 chrX chrY`

```

Comment:
Actually, for each chromosome we have to add a `+ Insert IChromosomes to include (and order to plot in):`. To do it in the described way would lead an error.

Reply:
The chromosomes are now: `chr2L chr2R chr3L chr3R chrX`